### PR TITLE
PREAPPS-1927: Read Receipts | Always/Never responses to Read Receipts

### PR DESCRIPTION
**Pull request by yashrajpchavda**
- Add mutation for SendDeliveryReport

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -822,6 +822,15 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(normalize(SearchResponse));
 
+	public sendDeliveryReport = (messageId: string) => {
+		return this.jsonRequest({
+			name: 'SendDeliveryReport',
+			body: {
+				mid: messageId
```
Comment (silentsakky) [inline src/batch-client/index.ts]: this conversion should happen using denormalization

Comment (SantoshKShetty) [reply src/batch-client/index.ts]: Done.

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -822,6 +824,14 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(normalize(SearchResponse));
 
+	public sendDeliveryReport = (message: SendDeliveryReportInput) =>
+		this.jsonRequest({
+			name: 'SendDeliveryReport',
+			body: {
```
Comment (lavegupta) [inline src/batch-client/index.ts]: body: denormalize(SendDeliveryReportInfo)(message)

Comment (SantoshKShetty) [reply src/batch-client/index.ts]: Done. Thanks

**Pull request merged**